commands/deploy.go

```diff
@@ -211,8 +212,10 @@ Error: %s`, fprocessErr.Error())
 				Limits:   function.Limits,
 				Requests: function.Requests,
 			}
-
-			proxy.DeployFunction(function.FProcess, services.Provider.GatewayURL, function.Name, function.Image, function.RegistryAuth, function.Language, deployFlags.replace, allEnvironment, services.Provider.Network, functionConstraints, deployFlags.update, deployFlags.secrets, allLabels, functionResourceRequest1)
+			statusCode := proxy.DeployFunction(function.FProcess, services.Provider.GatewayURL, function.Name, function.Image, function.RegistryAuth, function.Language, deployFlags.replace, allEnvironment, services.Provider.Network, functionConstraints, deployFlags.update, deployFlags.secrets, allLabels, functionResourceRequest1)
+			if statusCode != http.StatusOK && statusCode != http.StatusAccepted {
```

Comment:
Since we have the same check twice I'd be tempted to extract a function for use here and at line 237.
```
if checkForFailure(statusCode) {
...
}
```

then we can capture the functionality in one place:

``` func checkForFailure( statusCode int) bool{
return statusCode != http.StatusOK && statusCode != http.StatusAccepted
}
```